changelog/884.bugfix

```diff
@@ -0,0 +1 @@
+Fix hang caused by `steal` command with empty test queue (`worksteal` scheduler).
```

Comment:
```suggestion
Fixed hang using ``worksteal`` scheduler using a high number of CPU cores.
```

Reply:
The rationale here is that changelogs are read by end-users which are not usually familiarized with the internal, so better use a language closer to the usage of the plugin.

Reply:
But "high number of CPU cores" has nothing to do with the issue. I can reproduce it with `-n 3`

Reply:
Ahh OK, feel free to reword it. :+1: